tests/admin_filters/models.py

```diff
@@ -46,8 +46,20 @@ def __str__(self):
         return self.description
 
 
+class Office(models.Model):
+    code = models.CharField(max_length=3, unique=True)
+    name = models.CharField(max_length=50)
+
+    class Meta:
+        ordering = ('-pk',)
+
+    def __str__(self):
+        return self.name
```

Comment:
`__str__()` is not required, but if you want to use it in tests then use `name` attribute instead of `str(...)`, e.g.
```python
expected = [
    (self.black_office.pk, self.black_office.name),
    (self.blue_office.pk, self.blue_office.name),
    (self.red_office.pk, self.red_office.name),
]
```